pandas/core/strings.py

```diff
@@ -675,20 +675,42 @@ def str_replace(arr, pat, repl, n=-1, case=None, flags=0, regex=True):
 
 def str_repeat(arr, repeats):
     """
-    Duplicate each string in the Series/Index by indicated number
-    of times.
+    Duplicate each string in the Series or Index.
 
     Parameters
     ----------
-    repeats : int or array
-        Same value for all (int) or different value per (array)
+    repeats : int or sequence of int
```

Comment:
`sequence of int` might be fine, and maybe 'sequence' is a term that has a clear definition in the pandas community, but I often see `array_like` used in the numpy documentation. Not really recommending anything, just curious is `sequence` is used similarly here, or you were just using to to indicate a list/array/tuple/etc of integers.